lib/resources/security_policy.rb

```diff
@@ -74,8 +74,16 @@ class SecurityPolicy < Inspec.resource(1)
       describe security_policy do
         its('SeNetworkLogonRight') { should include 'S-1-5-11' }
       end
+
+      describe security_policy(translate_sid: true) do
+        its('SeNetworkLogonRight') { should include 'NT AUTHORITY\\Authenticated Users' }
+      end
     "
 
+    def initialize(pars = {})
```

Comment:
To be consistent with other resources, can you change this from `pars` to `opts`?